tests/util.py

```diff
@@ -160,3 +160,18 @@ def generate_path(dirname, filename, encoding=None):
     if encoding:
         path = path.encode(encoding)
     return path
+
+
+def sanlock_is_running():
+    exe_file = "./src/sanlock"
+    pid_file = "/tmp/sanlock/sanlock.pid"
+    try:
+        with open(pid_file, "r") as f:
+            for line in f:
```

Comment:
pid file contain exactly one line, so we should do:

    pid = f.readline().strip()